addons/calendar/static/src/js/base_calendar.js

```diff
@@ -8,7 +8,7 @@ var relationalFields = require('web.relational_fields');
 var session = require('web.session');
```

Comment:
Please, squash the commits and write a suitable message

Reply:
@Polymorphe57 Each and every commit fixes different issue, so I kept separate commit, did not squash.

Reply:
I don't see the point. The three last commits should really be part of the first one. Your task is to make an IMP in master not a series of fixes. I don't think it would make sense to merger the commits separately